.github/workflows/build.yml

```
@@ -0,0 +1,41 @@
+name: Build
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

When the `ubuntu-latest` version is updated this has a chance of suddenly breaking the build. I would recommend using a specific version to avoid that scenario.
```suggestion
    runs-on: ubuntu-18.04
```